src/resources/profile_features.jsx

```diff
@@ -0,0 +1,80 @@
+import { 
```

Comment:
I think this file is misnamed, it should use camelCase instead

Reply:
i'll change that